pyproject.toml

```diff
@@ -4,6 +4,7 @@ requires = ["setuptools", "wheel"]  # PEP 508 specifications.
 
 [tool.black]
 skip-string-normalization = false
+target-version = ["py311"]
```

Comment:
Does this mean that if when apply Black to Darker's own code base, it might become incompatible with Python versions older than 3.11?

Reply:
I'm don't think so, because 3.x python versions are backward compatible (as far as I know). What this does do is explicitly state we want support for newer constructs (like `match`).